packages/react-core/src/components/Alert/examples/Alert.md

```diff
@@ -64,8 +64,8 @@ import { Alert, AlertActionCloseButton, AlertActionLink } from '@patternfly/reac
     actionClose={<AlertActionCloseButton onClose={() => alert('Clicked the close button')} />}
     actionLinks={
       <React.Fragment>
-        <AlertActionLink onClick={() => alert('Clicked on View details')}>View details</AlertActionLink>
-        <AlertActionLink onClick={() => alert('Clicked on Ignore')}>Ignore</AlertActionLink>
+        <AlertActionLink component="a" href="#">View details</AlertActionLink>
+        <AlertActionLink component="a" href="#">Ignore</AlertActionLink>
```

Comment:
Good point, I did not realize at first that the "Ignore" is not a link but rather a close action.

- regarding replacing the alert popup with just a console log: I cannot use `console.log` because of the linter, so I left the action empty: `onClose={() => {}}`
- I removed the 3rd example and only kept the close button in the one example, where we explicitly point out the usage of `actionClose`